client/src/utils/functions.js

```diff
@@ -6,3 +6,11 @@ export function toIso(europeanDate) {
 export function formattedNumber(number) {
   return new Intl.NumberFormat("fr-FR").format(number);
 }
+
+export function categoryPercentage(categoryAmount, totalAmount) {
+  if (categoryAmount === 0 || totalAmount === 0) {
+    return 0;
+  }
+
+  return ((categoryAmount / totalAmount) * 100).toFixed(1);
+}
```

Comment:
**question**: I don't see anything related to a category, it just compute a percentage right ?